examples/macro/mocha.bzl

```diff
@@ -20,8 +20,8 @@ def mocha_test(name, srcs, args = [], data = [], env = {}, **kwargs):
         data = data + [
             "_%s_srcs" % name,
             "//examples/macro:mocha_reporters.json",
-            "@npm//examples/macro/mocha-multi-reporters",
-            "@npm//examples/macro/mocha-junit-reporter",
+            "//examples/macro:node_modules/mocha-multi-reporters",
```

Comment:
the ordering is so much more clear with the 1p package coming first and the node_modules address second 😍